markdown/cmark/Makefile.am

```diff
@@ -0,0 +1,43 @@
+if MD_BUILTIN_CMARK
+
+noinst_LTLIBRARIES = libcmark.la
+
+libcmark_la_SOURCES = \
+	blocks.c \
+	buffer.c \
+	buffer.h \
+	chunk.h \
+	cmark.c \
+	cmark_ctype.c \
+	cmark_ctype.h \
+	cmark.h \
+	commonmark.c \
+	houdini.h \
+	houdini_href_e.c \
+	houdini_html_e.c \
+	houdini_html_u.c \
+	html.c \
+	inlines.c \
+	inlines.h \
+	iterator.c \
+	iterator.h \
+	latex.c \
+	man.c \
+	node.c \
+	node.h \
+	parser.h \
+	references.c \
+	references.h \
+	render.c \
+	render.h \
+	scanners.c \
+	scanners.h \
+	utf8.c \
+	utf8.h \
+	xml.c
+
+EXTRA_DIST = \
+	case_fold_switch.inc \
+	entities.inc
```

Comment:
you also miss *scanners.re*.

Reply:
Oops, I meant to delete that file and just use the pre-generated code from upstream.